data/api/v1/full_spec.yaml

```diff
@@ -5677,11 +5677,24 @@ components:
           description: The name of the index.
           example: main
           type: string
+        num_flex_logs_retention_days:
+          description: 'The total number of days logs are stored in Standard and Flex
+            Tier before being deleted from the index.
+
+            If Standard Tier is enabled on this index, logs are first retained in
+            Standard Tier for the number of days specified through `num_retention_days`,
+
+            and then stored in Flex Tier until the number of days specified in `num_flex_logs_retention_days`
+            is reached.
+
+            The available values depend on the retention plans specified in your organization''s
```

Comment:
```suggestion
            The available values depend on the retention plans specified in your organization's
```

Reply:
I think this is fine, because we need to escape the `'` with another `'`. 
It's done like this already line 5690